src/solc_0.8/polygon/child/ChildLandBaseToken.sol

```diff
@@ -0,0 +1,85 @@
+    uint256 internal constant GRID_SIZE = 408;
+    // @review DO we need this?
+    uint256 internal constant LAYER = 0xFF00000000000000000000000000000000000000000000000000000000000000;
+
+    mapping(address => bool) internal _minters;
+    event Minter(address superOperator, bool enabled);
+
+    constructor(
+        address trustedForwarder,
+        uint8 chainIndex,
+        address admin
+    ) {
+        _admin = admin;
+        ERC721BaseToken.__ERC721BaseToken_initialize(chainIndex);
+        ERC2771Handler.__ERC2771Handler_initialize(trustedForwarder);
+    }
+
+    /// @notice Enable or disable the ability of `minter` to mint tokens
+    /// @param minter address that will be given/removed minter right.
+    /// @param enabled set whether the minter is enabled or disabled.
+    function setMinter(address minter, bool enabled) external {
+        require(msg.sender == _admin, "only admin is allowed to add minters");
+        _minters[minter] = enabled;
+        emit Minter(minter, enabled);
+    }
+
+    /// @notice check whether address `who` is given minter rights.
+    /// @param who The address to query.
+    /// @return whether the address has minter rights.
+    function isMinter(address who) public view returns (bool) {
+        return _minters[who];
+    }
+
+    /// @notice total width of the map
+    /// @return width
+    function width() external pure returns (uint256) {
+        return GRID_SIZE;
+    }
+
+    /// @notice total height of the map
+    /// @return height
+    function height() external pure returns (uint256) {
+        return GRID_SIZE;
+    }
+
+    /// @notice x coordinate of Land token
+    /// @param id tokenId
+    /// @return the x coordinates
+    function x(uint256 id) external view returns (uint256) {
+        require(_ownerOf(id) != address(0), "token does not exist");
+        return id % GRID_SIZE;
+    }
+
+    /// @notice y coordinate of Land token
+    /// @param id tokenId
+    /// @return the y coordinates
+    function y(uint256 id) external view returns (uint256) {
+        require(_ownerOf(id) != address(0), "token does not exist");
+        return id / GRID_SIZE;
+    }
+
+    // function _ownerOf(uint256 id) internal view returns (address) {
+    //     require(id & LAYER == 0, "Invalid token id");
+    //     uint256 packedOwnerData = _owners[id];
+    //     require(packedOwnerData != 0, "NON_EXISTANT_TOKEN");
+    //     return address(uint160(packedOwnerData)); // cast to zero
+    // }
+
+    // function _ownerAndOperatorEnabledOf(uint256 id) internal view returns (address owner, bool operatorEnabled) {
+    //     // @review Do we need this check?
+    //     require(id & LAYER == 0, "Invalid token id");
+    //     uint256 packedOwnerData = _owners[id];
+    //     require(packedOwnerData != 0, "NON_EXISTANT_TOKEN");
+    //     owner = address(uint160(packedOwnerData));
+    //     operatorEnabled = (packedOwnerData / 2**255) == 1;
+    // }
+}
```

Comment:
SHould be able to remove these too. They override the base class which should be sufficient here.